src/xenia/patcher/patcher.cc

```diff
@@ -0,0 +1,67 @@
+/**
+ ******************************************************************************
+ * Xenia : Xbox 360 Emulator Research Project                                 *
+ ******************************************************************************
+ * Copyright 2022 Ben Vanik. All rights reserved.                             *
+ * Released under the BSD license - see LICENSE in the root for more details. *
+ ******************************************************************************
+ */
+#include "xenia/patcher/patcher.h"
+
+#include "xenia/base/logging.h"
+
+namespace xe {
+namespace patcher {
+
+Patcher::Patcher(const std::filesystem::path patches_root) {
+  is_any_patch_applied_ = false;
+  patch_db = new PatchDB(patches_root);
+}
+
+Patcher::~Patcher() {}
```

Comment:
Unless there are issues with unique_ptr deletion, it should be either not declared at all, or defined as = default (though = default may be needed in the .cc rather than .h in some unique_ptr usage cases, it's somewhat confusing).